components/Publish.vue

```diff
@@ -74,7 +74,7 @@ input {
         </label>
         <label>
           <span>Image</span>
-          <input type="text" v-model="ad.image" placeholder="https://...." />
+          <input type="text" @change="checkImage" v-model="ad.image" placeholder="https://...." />
```

Comment:
I feel like this should be the `onLoad` of the preview image - somehow I feel like doing it on change here may have some race condition where this triggers before the image is loaded?

Reply:
Yea that would be ideal but the image lives inside a different component and that component is loaded 1621 times and I didn't want to add too much junk to it just to do this.

Ideally we'd refactor things to do better.